### PR TITLE
Added New Examples Page, Issue: #1294

### PR DESCRIPTION
**Pull request by Arminta-Jenkins-NIST**
# Committer Notes

We created a page similar to [OSCAL tools] (https://pages.nist.gov/OSCAL/tools/) that provides links to OSCAL content (catalogs, profiles, component-definitions, etc.) generated by NIST and website where community can contribute. 

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [x] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

File: docs/content/examples/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    weight: 50
+toc:
+  enabled: true
+---
+ 
+To facilitate the OSCAL adoption in ways that allow the adopters to take full advange of OSCAL abilities of representing the security data and the automation of the security assessment and of the risk management with a maximum return on investment (ROI), the NIST OSCAL team published basic examples of OSCAL content instances in the NIST's OSCAL-content repository, the [examples](https://github.com/usnistgov/oscal-content/tree/main/examples) directory.
+
+To properly understand the examples, interested parties are advised to also review the [OSCAL walkthrough tutorials](https://pages.nist.gov/OSCAL/learn/tutorials).
+
+{{% callout note %}} More OSCAL examples supporting the use of the OSCAL models are developed and maintained by the community in their [OSCAL Club](https://github.com/oscal-club/examples) repository.{{% /callout %}}
```
Comment (aj-stein-nist) [inline docs/content/examples/_index.md:16]: A minor change if you will accept it before approval.

```suggestion
{{% callout note %}} More OSCAL examples supporting the use of the OSCAL models are developed and maintained by the community in the [OSCAL Club examples repository](https://github.com/oscal-club/examples) .{{% /callout %}}
```

**Review by aj-stein-nist — CHANGES_REQUESTED** [review]
I would request a minor change, but other than that it looks good. Thanks so much to you and Michaela for working on this.

**Review by iMichaela — DISMISSED** [review]
I like AJ's suggestions. LGTM. We might want to also consider adding a link to the SP 800-53 catalogs and baselines on this page, but we can do it with a new PR if the team finds acceptable the proposal.

**Review by aj-stein-nist — APPROVED** [review]
LGTM, thanks to both of you.

**Comment by aj-stein-nist** [comment]
> I like AJ's suggestions. LGTM. We might want to also consider adding a link to the SP 800-53 catalogs and baselines on this page, but we can do it with a new PR if the team finds acceptable the proposal.

I did not miss the feedback in this comment, I am creating an issue to follow on with that to prioritize at a later date. Then we can close out the related issue for this.

**Pull request merged**